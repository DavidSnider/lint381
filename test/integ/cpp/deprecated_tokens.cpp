int* foo = NULL;

int* bar = malloc(5);

free(bar);

typedef foo bar;
