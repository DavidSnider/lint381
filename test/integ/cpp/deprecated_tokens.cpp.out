deprecated_tokens.cpp:1:12: error: Use 'nullptr' in C++ code, not 'NULL'
int* foo = NULL;
           ^^^^
deprecated_tokens.cpp:3:12: error: Use 'new' in C++ code, not 'malloc'
int* bar = malloc(5);
           ^^^^^^
deprecated_tokens.cpp:5:1: error: Use 'delete' in C++ code, not 'free'
free(bar);
^^^^
deprecated_tokens.cpp:7:1: error: Use 'using' in C++ code, not 'typedef'
typedef foo bar;
^^^^^^^
